aldryn_newsblog/utils/utilities.py

```diff
@@ -61,7 +61,8 @@ def strip_tags(value):
     strip tags.
     """
     # strip any new lines
-    value = value.strip()
+    if value:
+        value = value.strip()
 
```

Comment:
that wouldn't help if val was an empty string with spaces

```python
> bool(' ')
True
```

we _could_ add a third conditional but I think in general ``value`` should not be None/empty (=> evaluate to false) so I think it's better to just keep those two.